### PR TITLE
Fix craft helpfile

### PR DESCRIPTION
**Pull request by seanohue**
Also did some refactoring along the way with the help command and rendering of helpfiles/error handling.

### CODE REVIEW

**Comment by seanohue** [comment]
Resolves #239

File: bundles/ranvier-crafting/help/craft.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-command: quest
+command: craft <list/create> [category #] [item #]
```
Comment (shawncplus) [inline bundles/ranvier-crafting/help/craft.yml]: This is the wrong way to do it, the purpose of the `command` entry in the helpfile is to do a `CommandManager` lookup to get the `Command.usage` property. So this should not be necessary

File: bundles/ranvier-commands/commands/help.js [hunk 2]
```diff
@@ -40,7 +45,7 @@ module.exports = (srcPath) => {
 
     const formatHeaderItem = (item, value) => `${item}: ${value}\r\n\r\n`;
     if (hfile.command) {
-      let actualCommand = state.CommandManager.get(hfile.command);
+      let actualCommand = state.CommandManager.get(hfile.command) || state.CommandManager.get(hfile.command.split(' ')[0]);
```
Comment (shawncplus) [inline bundles/ranvier-commands/commands/help.js]: See comment below about `Command.usage`

File: bundles/ranvier-crafting/help/craft.yml (hunk 1 above)
Comment (seanohue) [inline bundles/ranvier-crafting/help/craft.yml]: Fixed.

**Comment by shawncplus** [comment]
:+1:

**Pull request merged**